src/metabase/models/interface.clj

```diff
@@ -443,6 +454,14 @@
   (-> instance
       add-entity-id))
 
+;; --- helper fns
+(defn pre-update-changes
```

Comment:
this function will makes the input for `before-update` match with what toucan1 currently returned. 

see https://github.com/camsaul/toucan2/blob/e2547454b5a5b808ca38ac9dfb264a07d2848676/toucan1/src/toucan/models.clj#L407-L426